examples/with-yoga/pages/_app.js

```diff
@@ -0,0 +1,22 @@
+import { ThemeProvider, FontLoader } from '@gympass/yoga'
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
```

Comment:
```suggestion
    margin: 20px;
```
Is there a specific requirement for `margin: 0` here or can we use a different value so the text isn't smooshed to the top?